.eslintrc.js

```diff
@@ -1,24 +1,29 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-const { getBaseConfig } = require('@edx/frontend-build');
+const { createConfig } = require('@edx/frontend-build');
 
-const config = getBaseConfig('eslint');
-
-config.overrides = [
-  {
-    files: ['*.test.js', '*.test.jsx'],
-    rules: {
-      'react/prop-types': 'off',
-      'react/jsx-no-constructed-context-values': 'off',
+const config = createConfig('eslint', {
```

Comment:
[note] Refactors to use `createConfig` instead of `getBaseConfig`, slightly more simplified as a result.